Observer/Config/Save.php

```diff
@@ -155,13 +168,35 @@ private function resetStoreCredentials($scopeId = null, $scope = ScopeInterface:
      * @param array <string> $changedPaths
      * @return bool
      */
-    public function isYotpoKeysChanged($changedPaths = [])
+    public function isYotpoSettingsChanged($changedPaths = [])
     {
         $yotpoKeyPaths = ['app_key', 'secret', 'yotpo_active'];
+        $commonPaths = $this->getChangedYotpoPaths($changedPaths, $yotpoKeyPaths);
+        return (bool)$commonPaths;
+    }
+
+    /**
+     * @param array <string> $changedPaths
+     * @return bool
+     */
+    public function isYotpoKeysChanged($changedPaths = [])
+    {
+        $yotpoKeyPaths = ['app_key'];
+        $commonPaths = $this->getChangedYotpoPaths($changedPaths, $yotpoKeyPaths);
+        return (bool)$commonPaths;
+    }
+
+    /**
+     * @param array <string> $changedPaths
+     * @param array <string> $pathsToCompare
+     * @return array <mixed>
+     */
+    public function getChangedYotpoPaths($changedPaths, $pathsToCompare)
```

Comment:
Please rename the second argument
```suggestion
    public function getChangedYotpoPaths($changedPaths, $keyPathsToCompare)
```

Reply:
This is done